dmn-repo/risk_check_en.dmn

```diff
@@ -0,0 +1,134 @@
+        <description></description>
+        <inputEntry id="UnaryTests_1dkokb9">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0d5r6v3">
+          <text>"Unemployed"</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1v6fey5">
+          <text>not("Basic Package")</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1rw2hjn">
+          <text>&lt;=95</text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_18kpqx8">
+          <text>"Lack Of Income"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_0mrbkzr">
+          <text>"red"</text>
+        </outputEntry>
+      </rule>
+      <rule id="DecisionRule_0c99hq3">
+        <description></description>
+        <inputEntry id="UnaryTests_16q1ghi">
+          <text>&lt;= 30</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0c86j25">
+          <text>"Self-employed"</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0l8wmca">
+          <text>"Premium Package"</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1qkgqif">
+          <text></text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_0jlpe55">
+          <text>"Irregular And Possibly Lack Of Income"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_05qilq2">
+          <text>"yellow"</text>
+        </outputEntry>
+      </rule>
+      <rule id="DecisionRule_1euaw2y">
+        <description></description>
+        <inputEntry id="UnaryTests_0bctfm4">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_12o8otd">
+          <text>"Freelancer", "Self-employed"</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0o842fe">
+          <text>-</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1h8e7id">
+          <text>&lt;=95</text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_1pwrw8z">
+          <text>"Not Trustworthy"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_0zcjpew">
+          <text>"yellow"</text>
+        </outputEntry>
+      </rule>
+      <rule id="DecisionRule_08nz8pq">
+        <description></description>
+        <inputEntry id="UnaryTests_0adgp9c">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1jzpweh">
+          <text>"Salaried"</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_17itih0">
+          <text>"Premium Package"</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0dgmj4d">
+          <text>&lt;90</text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_1xwbye7">
+          <text>"Too Low Income"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_1s5g29d">
+          <text>"yellow"</text>
+        </outputEntry>
+      </rule>
+    </decisionTable>
+  </decision>
+</definitions>
```

Comment:
@ikheraj17 why do you want to add an example DMN to the repository?

Reply:
So that users immediately see the folder in which files have to go and also have a working example in case they are new to DMN.

Reply:
@saig0 Are there any other outstanding issues regarding my pull request? If not, would you or the appropriate person please merge the changes that Falko and I have made? I'm in the process of creating documentation for the Camunda Cloud showcase, and the showcase's DMN functionality depends on this plugin. Please let me know if there are any other changes that you have questions about. Thank you!

Reply:
@ikheraj17 I'm sorry for the delay. I was distracted by other duties :see_no_evil: 

Currently, the build of this PR is [failing](https://github.com/camunda-community-hub/zeebe-dmn-worker/runs/4161189776?check_suite_focus=true#step:5:453). Something went wrong in the test case: `WorkflowTest.shouldCompleteWorkflowInstance()`. 

```
Error:  Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 34.072 s <<< FAILURE! - in io.zeebe.dmn.WorkflowTest
Error:  shouldCompleteWorkflowInstance  Time elapsed: 10.407 s  <<< ERROR!
``` 

Are you able to build the PR locally? 
I'll look into it today or tomorrow.

Reply:
I can reproduce the test error locally. Looking into it...